downstream/modules/platform/con-aap-upgrade-from-21.adoc

```diff
@@ -1,11 +1,15 @@
 
 
-[id="aap-upgrade-from-21_{context}"]
+[id="aap-upgrades_{context}"]
 
-= Upgrading from {PlatformNameShort} 2.1
+= {PlatformNameShort} upgrades
```

Comment:
Looking at the generated HTML there are other titles in this assembly that I think we should align to if you want to go ahead with this change. I didn't edit them in my commit because I didn't want to blow up the scope of your PR.